srcpkgs/puyovs/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'puyovs'
+pkgname=puyovs
+version=2022.11.28
+revision=1
+build_wrksrc=${pkgname}-nightly
```

Comment:
Highly doubtful that building a nightly version (that upstream even tells you not to use for ordinary usage) is suitable for packaging.

Reply:
We won't accept the package unless it is versioned appropriately.

Reply:
I'll wait for the next release then, since the last one had too many issues.